src/libs/utils.js

```diff
@@ -46,6 +47,23 @@ export const sleep = (milliseconds) => {
   return new Promise(resolve => setTimeout(resolve, milliseconds));
 };
 
+export const getDatasetNames = (datasets) => {
+  if(!datasets){return '';}
+  const datasetNames = datasets.map((dataset) => {
+    return ((dataset.label) ?  dataset.label : dataset.name);
```

Comment:
for some reason the datasets from chair console datasets have names stored as 'label' while datasets from admin and member have names stored as 'name'

Reply:
Could you dig into this a little more? In other words, there had to be a case where `name` was correct. What's different about the `datasets` argument from the other calls to this method? The source of the bug might be in the caller, not here.

Reply:
so when name is present, name is always correct, when label is present, label is always correct. The datasets returned from the api call DAR.getDataAccessManageV2() have labels while the datasets returned by DAR.getDarModalSummary(dar.dataRequestId) have names, I'm not sure this is so much a bug as it is a discrepancy, my first attempt was to get all the pages using the same api calls since they all display similar info, but because the pages are all structured so differently this required to changes to every single component used on the pages and I ended up scrapping that approach.

Reply:
so when name is present, name is always correct, when label is present, label is always correct. The datasets returned from the api call DAR.getDataAccessManageV2() have labels while the datasets returned by DAR.getDarModalSummary(dar.dataRequestId) have names, I'm not sure this is so much a bug as it is a discrepancy, my first attempt was to get all the pages using the same api calls since they all display similar info, but because the pages are all structured so differently this required to changes to every single component used on the pages and I ended up scrapping that approach.

Reply:
Ah, thank you - that makes sense.
1. `DAR.getDarModalSummary(dar.dataRequestId)` should be avoided and deprecated as soon as possible. The API is not reliable as it uses DTO style construction pulling from a variety of objects.
2. `DAR.getDataAccessManageV2()` is preferred as that gets the full objects instead of manually constructed value objects.